keel-artifact/keel-artifact.gradle.kts

```diff
@@ -9,7 +9,7 @@ dependencies {
   implementation(project(":keel-clouddriver"))
   implementation("org.springframework:spring-context")
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core")
-
+  implementation("com.netflix.spinnaker.kork:kork-artifacts")
```

Comment:
I thought you were trying to remove the dependency on `kork-artifacts`?

Reply:
Yes, this is only temporary while kork's Artifact coexists with the new SpinnakerArtifact in this PR.